src/main/java/javaslang/collection/Stream.java

```diff
@@ -1050,6 +1051,19 @@ default int length() {
     }
 
     @Override
+    default <U extends Comparable<? super U>> Stream<T> sortBy(Function<? super T, ? extends U> mapper) {
+        return sortBy(U::compareTo, mapper);
```

Comment:
Oh wow! I didn't know that method references could be applied to generic type variables! Sweet! :)